python/cog/director/__main__.py

```diff
@@ -49,12 +50,13 @@ def _die(signum: Any, frame: Any) -> None:
 
 events: queue.Queue = queue.Queue(maxsize=128)
 
-config = uvicorn.Config(create_app(events=events), port=4900, log_config=None)
+port = int(os.getenv("PORT", 4900))
```

Comment:
Is it weird that this is an environment variable while everything else is an argument? Should we make this an arg, too?